css/style.css

```diff
@@ -1,19 +1,205 @@
+/* Hero */
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+.hero {
+    background: url("../img/first-background.jpg");
+    height: 650px;
+    background-position: right 27% bottom 5%;
+    width: 100%;
+    flex-direction: column;
+}
+
+
+/* Hero Text */
+
+h1,
+.hero p {
+    color: white;
+}
+
+h1 {
+    font-size: 50px;
+    margin-bottom: 0;
+}
+
+.hero p {
+    font-size: 30px;
+}
+
+
+/* Karma Details */
+
+h2 {
+    font-size: 40px;
```

Comment:
See my previous comments on using rem for font-size. 

This is a great article explaining the difference and why we should use rem - https://www.24a11y.com/2019/pixels-vs-relative-units-in-css-why-its-still-a-big-deal/